tutorials/simulators/OpenFOAM.md

```diff
@@ -61,12 +43,29 @@ tutorial using the OpenFOAM Foundation distribution.
 :language: python
 ```
 
-## Example Code - ESI Distribution
+The current example is devided into four steps:
+
+1. **Machine Configuration**: In this step, we define the machine type and start
+it.
+
+2. **Download Input Files**: In this step, we download the input files from the
+inductiva bucket.
+
+3. **Picking the simulator**: In this step, we choose the simulator we want to
```

Comment:
**Pick the Simulator**: Here, we select the simulator we want to use. in this case, the OpenFOAM Foundation distribution.